sdk/examples/wallet/offline_signing/2_sign_transaction.rs

```diff
@@ -38,7 +41,8 @@ async fn main() -> Result<()> {
 
     // Signs prepared transaction offline.
     let unlocks = SecretManager::Stronghold(secret_manager)
-        .transaction_unlocks(&prepared_transaction_data)
+        // TODO don't use default protocol parameters
```

Comment:
What should we do here ? At least we should leave a comment explaining they should put some relevant parameters here

Reply:
We could store it together with the prepared transaction in the json file

Reply:
Added them in an own file